packages/dashboards/app/styles/navi-dashboards/components/navi-collection.less

```diff
@@ -55,6 +55,10 @@
 
     .action {
       margin-right: 10px;
+
+      .modal {
+        visibility: visible !important;
```

Comment:
Does this work? We didn't need `!important` in the directly styles:
```suggestion
        visibility: visible;
```

Reply:
without it it got overriden by the row visibility

Reply:
you might need to write a more specific selector

Reply:
targeted modal.is-active and its working now